packages/dataviews/src/dataviews.tsx

```diff
@@ -75,6 +76,7 @@ export default function DataViews< Item >( {
 	selection: selectionProperty,
 	setSelection: setSelectionProperty,
 	onSelectionChange = defaultOnSelectionChange,
+	header,
```

Comment:
This is a temporary prop until we address this issue https://github.com/WordPress/gutenberg/issues/63646